platform/ext/target/nordic_nrf/common/nrf5340/target_cfg.c

```diff
@@ -711,11 +716,19 @@ enum tfm_plat_err_t spu_periph_init_cfg(void)
      * - TWISx
      * - UARTEx
      */
+
+    /* When UART0 is a secure peripheral we need to leave Serial-Box 0 as Secure.
+     * The UART Driver will configure it as non-secure when it uninitializes.
+     */
+#if !(defined(SECURE_UART1) && NRF_SECURE_UART_INSTANCE == 0)
```

Comment:
If it's intentional that this says SECURE_UART1 then it should be explicitly documented why as without context it looks like a copy-paste error.